src/components/Header/SearchBar/SearchBar.tsx

```diff
@@ -103,30 +105,39 @@ export const SearchBar = ({
           'mx-24 mt-24 md:m-0': displayLocation !== 'homepage',
         })}
       >
-        <input
-          type="text"
-          ref={textInput}
-          placeholder="Search"
-          className={cn(searchInput)}
-          value={query}
-          onChange={handleSearch}
-          style={{ ...extraInputStyle }}
-        />
-        <Icon name="icon-gui-search" size="24px" additionalCSS="absolute left-16 top-12" />
-        {!isInFocus && (
-          <div className="absolute right-16 top-12 hidden lg:flex items-center justify-end">
-            <KeyIcon className="mr-4">{keyIcon}</KeyIcon>
-            <KeyIcon className="pt-2">K</KeyIcon>
-          </div>
+        {externalScriptsData.inkeepEnabled ? (
+          <InkeepSearchBar
+            className={`${cn(searchInput)} ${cn(searchInputNoPadding)}`}
+            extraInputStyle={{ ...extraInputStyle }}
+          />
+        ) : (
+          <>
+            <input
+              type="text"
+              ref={textInput}
+              placeholder="Search"
+              className={cn(searchInput)}
+              value={query}
+              onChange={handleSearch}
+              style={{ ...extraInputStyle }}
+            />
+            <Icon name="icon-gui-search" size="24px" additionalCSS="absolute left-16 top-12" />
+            {!isInFocus && (
+              <div className="absolute right-16 top-12 hidden lg:flex items-center justify-end">
+                <KeyIcon className="mr-4">{keyIcon}</KeyIcon>
+                <KeyIcon className="pt-2">K</KeyIcon>
+              </div>
+            )}
+            <SuggestionBox
+              results={results}
+              error={error}
+              query={query}
+              isActive={isInFocus}
+              displayLocation={String(displayLocation)}
```

Comment:
We shouldn't cast a potential `undefined` as a `String` here to satisfy the existing types of `SuggestionBox`, it undermines the benefit of what TS is providing, changes should be made elsewhere (I appreciate this isn't your code)

Reply:
Existing behaviour. Happy to look into outside of this PR but keen to minimise blast radius of changes